concepticondata/retired.json

```diff
@@ -28779,6 +28779,506 @@
+    },
+    {
+      "id": "Carling-2019-100-77",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-77"
+    },
+    {
+      "id": "Carling-2019-100-78",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-78"
+    },
+    {
+      "id": "Carling-2019-100-79",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-79"
+    },
+    {
+      "id": "Carling-2019-100-80",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-80"
+    },
+    {
+      "id": "Carling-2019-100-81",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-81"
+    },
+    {
+      "id": "Carling-2019-100-82",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-82"
+    },
+    {
+      "id": "Carling-2019-100-83",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-83"
+    },
+    {
+      "id": "Carling-2019-100-84",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-84"
+    },
+    {
+      "id": "Carling-2019-100-85",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-85"
+    },
+    {
+      "id": "Carling-2019-100-86",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-86"
+    },
+    {
+      "id": "Carling-2019-100-87",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-87"
+    },
+    {
+      "id": "Carling-2019-100-88",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-88"
+    },
+    {
+      "id": "Carling-2019-100-89",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-89"
+    },
+    {
+      "id": "Carling-2019-100-90",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-90"
+    },
+    {
+      "id": "Carling-2019-100-91",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-91"
+    },
+    {
+      "id": "Carling-2019-100-92",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-92"
+    },
+    {
+      "id": "Carling-2019-100-93",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-93"
+    },
+    {
+      "id": "Carling-2019-100-94",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-94"
+    },
+    {
+      "id": "Carling-2019-100-95",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-95"
+    },
+    {
+      "id": "Carling-2019-100-96",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-96"
+    },
+    {
+      "id": "Carling-2019-100-97",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-97"
+    },
+    {
+      "id": "Carling-2019-100-98",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-98"
+    },
+    {
+      "id": "Carling-2019-100-99",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-99"
+    },
+    {
+      "id": "Carling-2019-100-100",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-100"
```

Comment:
Has the policy of retiring concepts that are not in the list changed? I think I used `"replacement": null `once but as far as I can tell it's already outdated.

Reply:
Nope, I was just missing a commit. Thanks.